apps/web/src/views/Home/components/ColoredWordHeading.tsx

```diff
@@ -23,7 +25,7 @@ const ColoredWordHeading: React.FC<React.PropsWithChildren<HeadingProps>> = ({
   const displayedColor = (theme.colors[firstColor] as string) ?? theme.colors.textHome
 
   return (
-    <Heading scale="xxl" mb={mb} style={{ fontWeight: 800, maxWidth: '503px' }} {...props}>
+    <Heading scale="xxl" mb={mb} style={{ fontWeight: 800, maxWidth: '503px', ...style }} {...props}>
```

Comment:
Need to make sure to check if its defined, `...(style || {})`

Reply:
Fixed, now it will check if style is defined or not.